packages/cli/test/test-utils.js

```diff
@@ -49,96 +49,65 @@ exports.executeGenerator = function(GeneratorOrNamespace, settings) {
 };
 
 /**
- * Helper function for running the generator with custom inputs, artifacts,
- * and prompts.
- * @param {Generator} generator The generator to run.
- * @param {Object} prompts The prompts object to use with the generator under
- * test.
- * @param {Function} createArtifacts The create artifacts function. Takes
- * a directory. Use it to create folders and files for your tests.
- */
-exports.runGeneratorWith = function runGeneratorWith(
-  generator,
-  prompts,
-  createArtifacts,
-) {
-  return exports
-    .executeGenerator(generator)
-    .inTmpDir(dir => {
-      createArtifacts(dir);
-    })
-    .withPrompts(prompts);
-};
-/**
- * Setup the target directory with the required package.json and folder
- * structure.
- * @param {string} tmpDir Path to the temporary directory to setup
- * @param {object} options
- * @property {boolean} omitModelDir Do not create "models" directory
- * @property {boolean} omitRepositoryDir Do not create "repositories" directory
- * @property {boolean} omitControllerDir Do not create "controllers" directory
+ * Helper function for creating a LoopBack 4 Project Directory for
+ * testing.
+ *
+ * @param {string} rootDir Root directory in which to create the project
+ * @param {Object} options
+ * @property {boolean} excludeKeyword Excludes the 'loopback' keyword in package.json
+ * @property {boolean} excludePackageJSON Excludes package.json
+ * @property {boolean} excludeYoRcJSON Excludes .yo-rc.json
+ * @property {boolean} excludeControllersDir Excludes the controllers directory
+ * @property {boolean} excludeModelsDir Excludes the models directory
+ * @property {boolean} excludeRepositoriesDir Excludes the repositories directory
+ * @property {boolean} excludeDataSourcesDir Excludes the datasources directory
+ * @property {boolean} includeDummyModel Creates a dummy model file in /src/models/product-review.model.ts
+ * @property {boolean} includeDummyRepository Creates a dummy repository file in /src/repositories/bar.repository.ts
  */
-exports.givenAnApplicationDir = function(tmpDir, options) {
+exports.createLB4Project = function(rootDir, options) {
```

Comment:
Would you mind keeping "given" prefix? It comes from BDD, where arrange/act/assert is expressed as given/when/then.